RecoJets/JetProducers/plugins/FastjetJetProducer.cc

```diff
@@ -520,6 +455,97 @@ void FastjetJetProducer::runAlgorithm( edm::Event & iEvent, edm::EventSetup cons
+	descFastjetJetProducer.add<int>("maxDepth",	-1);
+	descFastjetJetProducer.add<int>("nFilt",	-1);
+	descFastjetJetProducer.add<int>("MinVtxNdof",	5);
+	///// From VirtualJetProducer
+	descFastjetJetProducer.add<string> ("@module_label",	"" );
+	descFastjetJetProducer.add<InputTag>("src",	InputTag("particleFlow") );
+	descFastjetJetProducer.add<bool>("doAreaFastjet",	false );
+	descFastjetJetProducer.add<double>("Rho_EtaMax", 	4.4 	);
+	descFastjetJetProducer.add<double>("rParam",		0.4 );
+	descFastjetJetProducer.add<string>("jetAlgorithm",	"AntiKt" );
+	descFastjetJetProducer.add<InputTag>("srcPVs",	InputTag("") );
+	descFastjetJetProducer.add<string>("jetType",		"PFJet" );
+	descFastjetJetProducer.add<double>("inputEtMin", 	0.0 );
+	descFastjetJetProducer.add<double>("inputEMin",		0.0 );
+	descFastjetJetProducer.add<double>("jetPtMin",		5. );
+	descFastjetJetProducer.add<bool>("doPVCorrection",	false );
+	descFastjetJetProducer.add<bool>("doRhoFastjet",	false );
+	descFastjetJetProducer.add<string>("jetCollInstanceName", ""	);
+	descFastjetJetProducer.add<bool>("doPUOffsetCorr", 	false	);
+	descFastjetJetProducer.add<string>("subtractorName", 	""	);
+	descFastjetJetProducer.add<bool>("useExplicitGhosts", 	false	);
+	descFastjetJetProducer.add<bool>("doAreaDiskApprox", 	false 	);
+	descFastjetJetProducer.add<double>("voronoiRfact", 	-0.9 	);
+	descFastjetJetProducer.add<double>("Ghost_EtaMax",	5. 	);
+	descFastjetJetProducer.add<int>("Active_Area_Repeats",	1 	);
+	descFastjetJetProducer.add<double>("GhostArea",	 	0.01 	);
+	descFastjetJetProducer.add<bool>("restrictInputs", 	false 	);
+	descFastjetJetProducer.add<unsigned int>("maxInputs", 	1 	);
+	descFastjetJetProducer.add<bool>("writeCompound", 	false 	);
+	descFastjetJetProducer.add<bool>("doFastJetNonUniform", false 	);
+	descFastjetJetProducer.add<bool>("useDeterministicSeed",false 	);
+	descFastjetJetProducer.add<unsigned int>("minSeed", 	14327 	);
+	descFastjetJetProducer.add<int>("verbosity", 		0 	);
+	descFastjetJetProducer.add<double>("puWidth",	 	0. 	);
+	descFastjetJetProducer.add<unsigned int>("nExclude", 	0 	);
+	descFastjetJetProducer.add<unsigned int>("maxBadEcalCells", 	9999999	);
+	descFastjetJetProducer.add<unsigned int>("maxBadHcalCells",	9999999 );
+	descFastjetJetProducer.add<unsigned int>("maxProblematicEcalCells",	9999999 );
+	descFastjetJetProducer.add<unsigned int>("maxProblematicHcalCells",	9999999 );
+	descFastjetJetProducer.add<unsigned int>("maxRecoveredEcalCells",	9999999 );
+	descFastjetJetProducer.add<unsigned int>("maxRecoveredHcalCells",	9999999 );
+	vector<double>  puCentersDefault;
+	descFastjetJetProducer.add<vector<double>>("puCenters", 	puCentersDefault);
+	///// From PileUpSubstractor
+	descFastjetJetProducer.add<double> ("puPtMin", 	10.);
+	descFastjetJetProducer.add<double> ("nSigmaPU", 	1.);
+	descFastjetJetProducer.add<double> ("radiusPU", 	0.5);
+	descFastjetJetProducer.add<bool> ("sumRecHits", 	false);
```

Comment:
This does not come from PileUpSubtractor (in this PR)...

In CMSSW_9_2_0, "sumRecHits" is only read by three HiJetAlgos modules:
- MultipleAlgoIterator
- ​ParametrizedSubtractor
- ​ReflectedIterator

They all derive from PileUpSubtractor, but this "sumRecHits" parameter is only defined in them (together with "dropZeroTowers"). It is not needed in this FastJetProducer, anyhow.

You had issues with removing this parameter in HLT with the customization function in the previous incarnation of this PR: did you try the suggestion of Martin in https://github.com/cms-sw/cmssw/pull/18832#issuecomment-303444805? Didn't it work?